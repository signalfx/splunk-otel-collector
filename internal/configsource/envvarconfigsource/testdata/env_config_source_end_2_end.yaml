config_sources:
  env:
    defaults:
      k0: 42
      cfg:
        k0: a string value
        k1: true

config:
  from_defined_env_var: ${env:_TEST_ENV_VAR_CFG_SRC}
  from_undefined_env_var: ${env:_UNDEFINED_ENV_VAR?optional=true}/some/path
  field_from_default: $env:k0
  map_from_default: $env:cfg?optional=true
