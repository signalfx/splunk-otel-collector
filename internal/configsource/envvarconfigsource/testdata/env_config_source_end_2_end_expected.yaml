config:
  from_defined_env_var: test_env_var
  from_undefined_env_var: /some/path
  field_from_default: 42
  map_from_default:
    k0: a string value
    k1: true

